bundles/admin/admin-layereditor/view/LayerHelper.js

```diff
@@ -0,0 +1,157 @@
+export const LayerHelper = (Oskari) => {
```

Comment:
Export LayerHelper as a class and leave private functions out of it.

Reply:
I'll rename it getLayerHelper and pass an array of supported languages instead of Oskari as param. It's mostly code moved from LayerFormService to a helper file to make the service file more readable. The service itself doesn't even use the new mutator mixin so this needs more work to be ready.